client/scss/components/_footer.scss

```diff
@@ -1,189 +1,112 @@
 @use 'sass:math';
 
 .footer {
```

Comment:
I’ve done a fair bit of refactoring here – not necessarily changing the styles too much but at least making sure to reduce selector specificity where possible. A lot of styles just moved further down the file so this is hard to review as a line-by-line diff.